src/core/grid_based_algorithms/lb.cpp

```diff
@@ -1344,6 +1344,15 @@ int lb_lbfluid_get_interpolated_velocity_global(Vector3d &p, double *v) {
 
   return 0;
 }
+#ifdef LB_GPU
+double lbgpu_eq_stress() {
+  double p0 = 0.;
+  for (int ii = 0; ii < LB_COMPONENTS; ii++) {
+    p0 += lbpar_gpu.rho[ii] * lbpar_gpu.agrid * lbpar_gpu.agrid /
```

Comment:
is ther any literature for this formula?

Reply:
I think the formula is correct [Ulf Schiller's lecture](http://espressomd.org/wordpress/wp-content/uploads/2017/10/summer_school_2017_lattice_boltzmann.pdf) on slide 38+. However, I ran through the slides and on slide 43 there is a formula for the stress observable as 0.5(f_i + f*_i).
The questions is if we want to supply the user with the average or with two observables and he has to do the post calculation on his own.